cumulus_library/databases/duckdb.py

```diff
@@ -207,6 +182,28 @@ def operational_errors(self) -> tuple[type[Exception], ...]:
             duckdb.BinderException,
         )
 
+    def export_table_as_parquet(
+        self, table_name: str, table_type: str, location: pathlib.Path, *args, **kwargs
+    ) -> str | None:
+        parquet_path = location / f"{table_name}.{table_type}.parquet"
+        parquet_path.parent.mkdir(exist_ok=True, parents=True)
+        table_size = self.connection.execute(f"SELECT count(*) FROM {table_name}").fetchone()  # noqa: S608
+        if table_size[0] == 0:
+            return None
+        query = f"""COPY
+            (SELECT * FROM {table_name})
+            TO '{parquet_path}'
+            (FORMAT parquet)
+            """  # noqa: S608
+        self.connection.execute(query)
+
+        df = pandas.read_parquet(parquet_path)
+        df = df.sort_values(
+            by=list(df.columns), ascending=False, ignore_index=True, na_position="first"
+        )
```

Comment:
nit: I think you could do this in the SQL above: `ORDER BY ALL DESC NULLS FIRST`

Reply:
If you don't want to have optimized-per-db versions of this ordering, maybe bring it up out of the db-specific code, so it's not duplicated?